### DISCUSSION TITLE
Can't figure out how to get verified

### DISCUSSION THREAD

**Discussion by leelrs** [community/community#134967] [GitHub Education]
### Hi everyone,

Hey there, tryin to get student verified for the student dev tools, but my uni-issued student ID does not have any sort of date on it, and the app doesn't seem to like me trying to submit a screenshot of my schedule. Not sure what to do from here, any help appreciated :)

**Comment by PakmanGames** [comment]
Request a proof of enrollment letter from your uni and make sure it has your full legal name and a date on it
